src/auframe/auframe.c

```diff
@@ -11,6 +11,10 @@
 #include <rem_auframe.h>
 
 
+enum {
+	SILENCE_Q = 1024 * 1024,  /* Quadratic sample value for silence */
```

Comment:
We can cleanup this right?

Reply:
Yes, I tested with G711, G722 and opus with different ptime and buffer settings.

We are testing this now in our products. But since it is by default set to FIXED for aubuf and jbuf nothing changes for a default setup. So, there is little risk.